prisma/schema.prisma

```diff
@@ -102,3 +103,12 @@ model EventImage {
   repairEvent   Event  @relation(fields: [repairEventId], references: [id])
   repairEventId String
 }
+
+model EventRepairer {
+  id      String @id @default(uuid())
+  userId  String
+  eventId String
```

Comment:
Is there a specific reason why these are strings instead of numbers?

Reply:
String is good but you need to add a foreign key reference to Event table.

Reply:
For future reference: 
- id is in the form of a UUID
- userId and eventId uses a Clerk string ID